runtime/cudaq/platform/default/rest/RemoteRESTQPU.cpp

```diff
@@ -230,10 +233,10 @@ class RemoteRESTQPU : public cudaq::QPU {
 
     // Loop through the file, extract the pass pipeline and CODEGEN Type
     auto lines = cudaq::split(configContents, '\n');
-    std::regex pipeline("PLATFORM_LOWERING_CONFIG\\s*=\\s*\"(\\S+)\"");
-    std::regex emissionType("CODEGEN_EMISSION\\s*=\\s*(\\S+)");
+    std::regex pipeline("^PLATFORM_LOWERING_CONFIG\\s*=\\s*\"(\\S+)\"");
+    std::regex emissionType("^CODEGEN_EMISSION\\s*=\\s*(\\S+)");
     std::smatch match;
-    for (const std::string &line : lines) {
+    for (std::string &line : lines) {
```

Comment:
Is `line` being modified?

Reply:
Ah, good catch. It was at one point, but I backed that change out in favor of another change, but I forgot to restore the `const`. Updated in bf41f69